testmodels/src/main/java/com/amplifyframework/testmodels/ratingsblog/Blog.java

```diff
@@ -38,6 +39,7 @@ public final class Blog implements Model {
     private final @ModelField(targetType="String", isRequired = true) String name;
     private final @ModelField(targetType="String") List<String> tags;
     private final @ModelField(targetType="Post") @HasMany(associatedWith = "blog", type = Post.class) List<Post> posts = null;
+
```

Comment:
Should a `resolveIdentifier` method be added here?

Reply:
We don't need to add resolveIdentifier, this demonstrates that this is backwards compatible. :)